src/linux/minidump_writer.rs

```diff
@@ -383,3 +450,13 @@ impl MinidumpWriter {
         Ok(section.location())
     }
 }
+
+fn write_soft_errors(
+    buffer: &mut DumpBuf,
+    soft_errors: SoftErrorList<WriterError>,
+) -> Result<MDLocationDescriptor> {
+    let soft_error_list_str = format!("{soft_errors:?}");
+
+    let section = MemoryArrayWriter::write_bytes(buffer, soft_error_list_str.as_bytes());
+    Ok(section.location())
+}
```

Comment:
I like that this is another stream

Reply:
😁😁😁 - Don't forget that I need to get that added to `minidump-common` before I can merge this!

Reply:
Not necessarily, rust-minidump will ignore unknown streams, so we can land this if we want to, it's just that it won't be usable until we add support to `minidump-common`

Reply:
Oh wait, the definition of the stream is in `minidump-common`, silly me.